extensions/ql-vscode/src/view/result-tables.tsx

```diff
@@ -345,8 +350,15 @@ class ResultTable extends React.Component<ResultTableProps, {}> {
     switch (resultSet.t) {
       case 'RawResultSet': return <RawTable
         {...this.props} resultSet={resultSet} />;
-      case 'SarifResultSet': return <PathTable
-        {...this.props} resultSet={resultSet} />;
+      case 'InterpretedResultSet': {
+        const data = resultSet.interpretation.data;
+        switch (data.t) {
+          case 'SarifInterpretationData': {
+            const sarifResultSet = { ...resultSet, interpretation: { ...resultSet.interpretation, data } };
```

Comment:
This is confusing to me. It doesn't look like anything is changing here. Could you add a comment to explain what you're doing?

Reply:
Right, this dispatch on `t` only makes sense when we have other interpretation types, so I can move it in to the other PR.